drivers/sensor/ti/bq274xx/bq274xx.c

```diff
@@ -418,7 +418,8 @@ static int bq274xx_gauge_configure(const struct device *dev)
 	}
 	regs = data->regs;
 
-	designenergy_mwh = (uint32_t)config->design_capacity * 37 / 10; /* x3.7 */
+	designenergy_mwh = (uint32_t)config->design_capacity *
+										((uint32_t)config->design_voltage) / 1000;
```

Comment:
ok now that's like double the tabs they need to be :-) set your editor (and github, who sadly changed the default) to 8 spaces per tab